DeepSpeech/import_lingualibre.sh

```diff
@@ -4,7 +4,7 @@ set -xe
 
 pushd $HOME/ds/
 	if [ ! -f "/mnt/sources/lingua_libre_Q385-ita-Italian_train.zip" ]; then
-		wget https://lingualibre.fr/datasets/Q385-ita-Italian.zip -O /mnt/source/lingua_libre_Q385-ita-Italian_train.zip
+		wget https://lingualibre.fr/datasets/Q385-ita-Italian.zip -O /mnt/sources/lingua_libre_Q385-ita-Italian_train.zip
```

Comment:
I don't know why you need all that code, `bin/import_lingua_libre.py` already does perform download.

Reply:
we didn't know and we added manually in the bash file.
We can remove it without any issue